pip/BUILD

```diff
@@ -44,4 +45,4 @@ py_binary(
     name = "repackage",
     srcs = ["repackage.py"],
     visibility = ["//visibility:public"]
-)
+)
```

Comment:
Please ensure newlines at the end of each file!

Reply:
addressed.